actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -112,6 +113,10 @@ private void updateUser(Request actorMessage) {
       isPrivate = (boolean) actorMessage.getContext().get(JsonKey.PRIVATE);
     }
     if (!isPrivate) {
+      actorMessage.getRequest().remove(JsonKey.ORGANISATIONS);
```

Comment:
probaably better to throw error - in such cases - when we are not able to perform the operation